profiles/scraper.py

```diff
@@ -533,22 +590,20 @@ def parse_profile_education(self):
                     education, "FIELD_OF_STUDY").split('\n')[1] if self.element_get_text_by_xpath_or_none(education, "FIELD_OF_STUDY") is not None else "" 
             degree = self.element_get_text_by_xpath_or_none(
                     education, "DEGREE").split('\n')[1] if self.element_get_text_by_xpath_or_none(education, "DEGREE") is not None else ""
-            
-            # degree = degree.lower()
 
-            # if "bachelor" in degree or "bcs" in degree or "BSC" in degree :
-            #     degree = "Bachelor"
-            # elif "master" in degree:
-            #     degree = "Master"
-            # elif "phd" in degree:
-            #     degree = "PhD"
-            # else:
-            #     degree = "Othe
```

Comment:
this was commented out because this doesn't cover enough edge cases, so its better to just save the raw data, this field is not used in the profile2vec component